light_node/src/main.rs

```diff
@@ -511,17 +512,22 @@ fn main() {
     );
 
     // initialize dbs
+    let maindb = initialize_maindb(
+        &log,
+        &env.storage.db_path,
+        env.storage.db.expected_db_version,
+        &main_chain,
+    )
+    .expect("Failed to initialize main db (sled) storage");
     let kv_cache = RocksDbCache::new_lru_cache(env.storage.db.cache_size)
```

Comment:
@mambisi 
this kv_cache should be just configured if ROCKSDB is selected,
so I would just add `caches`  parameter to the `initialize_maindb`

Reply:
RockDB was removed from possible backend because of its `Iterator` lifetime constraints

Reply:
@mambisi 
if we dont need rocksdb anymore, just remove it and also from dependencies, right?

Reply:
no we will need it for context_actions

Reply:
it still has configuration for rockdb as a backend